packages/grid/README.md

```diff
@@ -17,21 +17,67 @@ instead:
 yarn add @carbon/grid
 ```
 
-## How it works
-
-- Talk about flexbox and call out to how it works
-  - Also mention CSS Grid implementation?
-  - Talk about grid structure
-    - `.grid` > `.row` > `.col`
-- Talk about grid options/parameters in a table
-- Responsive classes
-- Nesting
-- Helpers
-  - Columns that auto-fill
-  - Alignment
-  - Ordering
-- SCSS documentation
-- Customizing the grid?
+## Usage
+
+`@carbon/grid` has three primitive class types to use in order to structure your
+application. They include:
+
+- `bx--grid`, defines the overall grid context and sets some useful attributes
+  like width and margin
+- `bx--row`, defines a row of items in a grid
+- `bx--col`, used to define individual columns
+
+You can use a combination of these classes to build a layout. For example, if we
+wanted a 4 column layout for a small breakpoint we could use the following
+markup:
+
+```html
+<div class="bx--grid">
+  <div class="bx--row">
+    <div class="bx--col">1/4</div>
```

Comment:
Correct me if I'm wrong, but I read further down that `bx--col` would be for "defin[ing] a column that spans as much width as available", rather than for a small breakpoint as the text on line 31 says?

Reply:
Good call, wasn't sure if the language here would be ambiguous or not. Will update!